dts/arm/st/h7/stm32h745.dtsi

```diff
@@ -44,6 +45,19 @@
 			status = "disabled";
 		};
 
+		mipi_dsi: dsihost@50000000 {
+			compatible = "st,stm32-mipi-dsi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x50000000 0x1000>;
+			clock-names = "dsiclk", "refclk", "pixelclk";
+			clocks = <&rcc STM32_CLOCK_BUS_APB3 0x00000010>,
+				 <&rcc STM32_SRC_HSE DSI_SEL(1)>,
+				 <&rcc STM32_SRC_PLL3_R DSI_SEL(1)>;
+			resets = <&rctl STM32_RESET(APB3, 4U)>;
+			status = "disabled";
+		};
+
```

Comment:
I think that has to go to `stm32h747.dtsi`.
747/757 have DSI, right?

Reply:
Correct, DSI is not supported on h745

Reply:
Moved to `stm32h747.dtsi`